cmd/antrea-agent/agent.go

```diff
@@ -167,6 +168,10 @@ func run(o *Options) error {
 	if o.config.ServiceCIDRv6 != "" {
 		_, serviceCIDRNetv6, _ = net.ParseCIDR(o.config.ServiceCIDRv6)
 	}
+	var serviceCIDRController *servicecidr.Controller
+	if o.config.AntreaProxy.ProxyAll {
```

Comment:
We already start serviceInformer unconditionally, there is no much cost to calculate serviceCIDR with the cache, and the CIDR are required by more and more modules, I think we could calculate it unconditionally.